src/traces/volume/defaults.js

```diff
@@ -0,0 +1,70 @@
+/**
+* Copyright 2012-2019, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var isosurfaceDefaults = require('../isosurface/defaults');
+var Lib = require('../../lib');
+var attributes = require('./attributes');
+
+var MIN = 0.1; // Note: often we don't want the data cube to be disappeared
+
+function createWave(n, minOpacity) {
+    var arr = [];
+    var steps = 32; // Max: 256
+    for(var i = 0; i < steps; i++) {
+        var u = i / (steps - 1);
+        var v = minOpacity + (1 - minOpacity) * (1 - Math.pow(Math.sin(n * u * Math.PI), 2));
+        arr.push([
+            u,
+            Math.max(1, Math.min(0, v))
+        ]);
+    }
+    return arr;
+}
+
+module.exports = function supplyDefaults(traceIn, traceOut, defaultColor, layout) {
+    function coerce(attr, dflt) {
+        return Lib.coerce(traceIn, traceOut, attributes, attr, dflt);
+    }
+
+    isosurfaceDefaults(traceIn, traceOut, defaultColor, layout);
```

Comment:
Hmm. I don't think we can do that. `isosurface` don't have similar attributes, but they have different `dflt` for `volume`, correct?

Reply:
There is only one attribute `spaceframe` that has different `dflt`.

Reply:
Ok, where is `spaceframe` getting coerced?

Reply:
Good catch!

Reply:
Addressed in https://github.com/plotly/plotly.js/pull/3488/commits/77f7317b40f922cadca5b2aa6609732c99b8bbab.